docs/src/adoc/index.adoc

```diff
@@ -1680,6 +1696,38 @@ handle.createUpdate("insert into groups (id, user_ids) values (:id, :users)")
 Like the <<Arguments Registry>>, if there are multiple `SqlArrayType` s
 registered for the same data type, the last registered wins.
 
+==== Mapping custom array types
+
+`SqlArrayType` only allows you to bind Java array/collection arguments to their
+SQL counterparts. To map SQL array columns back to Java types, you can register
+a regular `ColumnMapper`:
+
+[source,java]
+----
+public class UserIdListColumnMapper implements ColumnMapper<List<UserId>> {
```

Comment:
Actually, you only need to register a `ColumnMapper<UserId>` and Jdbi figures out the rest, for any type of collection that's registered with `JdbiCollectors` config class.

Reply:
thanks, fixed. Much clearer now